test/ILLink.RoslynAnalyzer.Tests/Verifiers/CSharpAnalyzerVerifier`1.cs

```diff
@@ -42,12 +42,15 @@ public static DiagnosticResult Diagnostic (DiagnosticDescriptor descriptor)
 			SyntaxTree src,
 			(string, string)[]? globalAnalyzerOptions = null)
 		{
+			TestCaseUtils.GetDirectoryPaths (out _, out string testAssemblyPath);
+			var expectationsPath = Path.Combine (testAssemblyPath, "Mono.Linker.Tests.Cases.Expectations.dll");
```

Comment:
This doesn't seem bad, but we could use `typeof(<some-type-we-know).Assembly.Location` here. We know it's safe because we wrote that assembly and know that it doesn't have a ref assembly, and using `typeof` would prevent this from failing by us not building the dependent assembly. If we can use a `typeof` then clearly the assembly was built and referenced.